tests/Resources/DataFixtures/Page/template_with_xinclude_properties.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" ?>
+<properties xmlns="http://schemas.sulu.io/template/template"
```

Comment:
I do not see what size has to do with effectiveness of the test, but I'm happy to change it if you insist